partiql-spi/src/main/kotlin/org/partiql/spi/connector/sql/builtins/AggCount.kt

```diff
@@ -13,7 +13,7 @@ import org.partiql.value.PartiQLValueType.ANY
 import org.partiql.value.PartiQLValueType.INT64
 
 @OptIn(PartiQLValueExperimental::class, FnExperimental::class)
-public object Agg_COUNT__ANY__INT32 : Agg {
+public object Agg_COUNT__ANY__INT64 : Agg {
```

Comment:
(self-review) I noticed the return type was changed to `INT64` in a recent PR but the name wasn't changed